modules/geolocation/geolocation.py

```diff
@@ -129,3 +130,71 @@ def calculate_pixel_to_geo_mapping(self):
 
         # Return matrix product of mappedGeoMatrix and sourcePixelMatrixInverse
         return (mappedGeoMatrix.dot(sourcePixelMatrixInverse))
+    
+class OutputAnalysis:
```

Comment:
@Xierumeng Did you want this to be a seperate class? I think get_best_location should just be in the geolocation class, this kind of goes against my architecture

Reply:
I wasn't too sure either, initially I attempted to merge them together in one class but I got errors, so I had placed it in as a separate class. I can work on merging them together as a single class this week if they need to be separate

Reply:
Should be inside the `Geolocation` class. What's the problem?